### ISSUE TITLE
Basic usage question: docs for functions/classes are not generated

### ISSUE THREAD

**Issue by stscieisenhamer** [astropy/sphinx-automodapi#17]
Probably just a basic usage quesion: The automodapi says that 

> For each function and class, a full API page will be generated.

The main module documentation is created, with the list of functions and classes. However, instead of generating any further documentation, I get a list of warnings saying that the doc pages do not exist:
```
api.rst: found 16 automodsumm entries to generate
building [mo]: targets for 0 po files that are out of date
building [html]: targets for 6 source files that are out of date
updating environment: 6 added, 0 changed, 0 removed
reading sources... [100%] viewer
/devdir/specviz/docs/source/api.rst:15: WARNING: toctree references unknown document 'api/specviz.analysis.centroid'
/devdir/specviz/docs/source/api.rst:15: WARNING: toctree references unknown document 'api/specviz.analysis.eq_width'
/devdir/specviz/docs/source/api.rst:15: WARNING: toctree references unknown document 'api/specviz.analysis.extract'
/devdir/specviz/docs/source/api.rst:15: WARNING: toctree references unknown document 'api/specviz.analysis.fwzi'
/devdir/specviz/docs/source/api.rst:15: WARNING: toctree references unknown document 'api/specviz.analysis.resample'
/devdir/specviz/docs/source/api.rst:15: WARNING: toctree references unknown document 'api/specviz.analysis.stats'
...
```

The snippet from the `app.rst` file is
```
Analysis Functions
------------------
.. automodapi:: specviz.analysis
  :no-heading:
```

**Comment by astrofrog** [comment]
Are you using the latest developer version of sphinx-automodapi? If not, can you try that?

If that doesn't help, can you try adding:

```
numpydoc_class_members_toctree = False
```

to your ``conf.py`` file?

**Comment by astrofrog** [comment]
Also just to check, is an ``api`` folder generated in either ``/devdir/specviz/docs`` or ``/devdir/specviz/docs/source`` and if so, what does it contain?

**Comment by stscieisenhamer** [comment]
An api folder is created under docs. However, if I remove the folder, without removing the _build folder, no errors are generated. If I remove the _build folder, the errors return.

Will try dev version and also the numpydoc_class_memeber_toctree setting.

**Comment by stscieisenhamer** [comment]
Using the lastest master and setting numpydoc_class_members_toctree makes no difference.

**Comment by astrofrog** [comment]
@stscieisenhamer - I'm not seeing these warnings with the latest developer version of sphinx-automodapi. Can you start from a clean clone of the specviz repo and check again? How are you building the docs? (I'm using ``python setup.py build_docs``)

**Comment by stscieisenhamer** [comment]
grrrr...i hate when magic happens. So, now I am no longer seeing any errors either. I had been using `make html` in the docs folder. But even that is working.

My only guess is that there was a undetermined interaction between another app I use while doing documentation called `sphinx-autobuild`. This app watches the docs folder for changes and does a rebuild when something changes. This morning I had disabled for some of the testing, and things have started working. I had not realized that sphinx-automodapi was creating rst files and this may have been confusing things with the autobuild task.

At least now I have a point to keep an eye on if it crops up again.

**Comment by astrofrog** [comment]
@stscieisenhamer - I've just released v0.2, so if you were needing to use the latest dev version, you can now switch to using v0.2.

**Comment by stscieisenhamer** [comment]
Will pull. Thanks!
________________________________
